projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -5432,9 +5431,20 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
 
         this.endRowTransaction(commit, row);
 
-        const currentCell = this.gridAPI.get_cell_by_index(ri, columnindex);
-        if (currentCell && event) {
-            currentCell.nativeElement.focus();
+        if (event) {
+            const currentCell = this.gridAPI.get_cell_by_index(ri, columnIndex);
```

Comment:
Slight optimization and clarification - drop this line in the check below and change that to look for cell instead. There's no reason to run get_cell with -1 index when you know there's no edit cell anyway.

Reply:
Nope, if you tab to row edit overlay buttons there is no edit cell. In this case `columnIndex` is -1 and `currentCell` is `undefined`. In this specific case we are falling back to `activeElement` of the selection service.